.github/workflows/infrastructure-pr.yml

```diff
@@ -0,0 +1,33 @@
+name: Trigger auto deployment in infrastructure repo
+
+on:
+  workflow_call:
+    inputs:
+      builder_digest:
+        required: true
+        type: string
+      runtime_digest:
+        required: true
+        type: string
+
+    secrets:
+      CI_PR_PAT:
+        description: "PAT for testing"
+        required: true
+
+jobs:
+  main:
+    name: Trigger workflow_dispatch event in infrastructure repo
+    runs-on: ubuntu-latest
+
+    steps:
+      # TODO: review Github CLI
+      - name: call workflow_dispatch API
+        run: |
+          curl -L \
```

Comment:
Thoughts on using `gh` to do this? Is it possible?

Reply:
That's what the TODO is for. Just this version for know as we know it works.